src/main/java/mate/academy/springboot/datajpa/dto/mapper/RequestDtoMapper.java

```diff
@@ -0,0 +1,5 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+public interface RequestDtoMapper<D, T> {
```

Comment:
```
public interface DtoMapper<M, D, R> {
    D toDto(M model);
    M toModel(R dto);
}
```
Instead of 2 interfaces, you can create one and use it